CMakeLists.txt

```
@@ -15,6 +15,11 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
 option(SNAP_BUILD "Building as a snap?" OFF)
 
+set(MIR_LIBRARIES_PKGCONFIG_DIRECTORY "" CACHE STRING "Search for Mir libraries pc files in this directory")
```

I like to keep all the options together :)

```suggestion
set(MIR_LIBRARIES_PKGCONFIG_DIRECTORY "" CACHE STRING "Search for Mir libraries pc files in this directory")

```

---

Ok, but see conversation below because option ordering is modified by last commit